test/demo/raiseIntent.test.ts

```diff
@@ -78,6 +80,12 @@ describe('Intent listeners and raising intents', () => {
                     const receivedContexts = await listener.getReceivedContexts();
                     expect(receivedContexts).toEqual([validPayload.context]);
                 });
+                test('When calling unsubscribe from the intent listener, then calling raiseIntent from another app, it times out', async () => {
+                    await listener.unsubscribe();
+                    const resultPromise = fdc3Remote.raiseIntent(testManagerIdentity, validPayload.intent, validPayload.context, testAppIdentity.appId);
+
+                    await expect(resultPromise).rejects.toThrowError(`Timeout waiting for intent listener to be added. intent = ${validPayload.intent}`);
+                }, INTENT_LISTENER_TIMEOUT * 2);
```

Comment:
What we really want here is INTENT_LISTENER_TIMEOUT + some small buffer, right? Would be happier if we did that (so something like `INTENT_LISTENER_TIMEOUT + 500`)

Reply:
done